pom.xml

```diff
@@ -103,19 +103,19 @@
     <!-- If updating dependencies, please update any relevant javadoc offlineLinks -->
     <avro.version>1.8.1</avro.version>
     <bigquery.version>v2-rev295-1.22.0</bigquery.version>
-    <pubsubgrpc.version>0.0.2</pubsubgrpc.version>
+    <pubsubgrpc.version>0.1.0</pubsubgrpc.version>
     <clouddebugger.version>v2-rev8-1.22.0</clouddebugger.version>
     <dataflow.version>v1b3-rev36-1.22.0</dataflow.version>
     <dataflow.proto.version>0.5.160222</dataflow.proto.version>
-    <datastore.client.version>1.1.0</datastore.client.version>
-    <datastore.proto.version>1.0.1</datastore.proto.version>
+    <datastore.client.version>1.2.0</datastore.client.version>
+    <datastore.proto.version>1.2.0</datastore.proto.version>
```

Comment:
its for the grpc/protobuf changes
those libraries depend on grpc 1.0.0+ and protobuf 3.0.0+